src/reporting.rs

```diff
@@ -0,0 +1,78 @@
+use anyhow::Result;
+use reqwest::Client;
+use tracing::debug;
+
+use crate::models::{
+    DiscordEmbed, DiscordEmbedAuthor, DiscordWebhookEvent, MemberCreator, WebhookActionDisplay,
+    WebhookEvent,
+};
+
+pub struct DiscordReporter {
+    endpoint: String,
+    client: Client,
+}
+
+impl DiscordReporter {
+    pub fn new(endpoint: String) -> Self {
+        Self {
+            endpoint,
+            client: Client::new(),
+        }
+    }
+
+    pub async fn report(&self, event: WebhookEvent) -> Result<()> {
+        if Self::should_skip(&event) {
+            debug!("Skipping event");
+            return Ok(());
+        }
+        let mut card_name: Option<String> = None;
+        let mut card_url: Option<String> = None;
+        if let Some(card) = event.action.data.card {
+            card_name = Some(card.name);
+            card_url = Some(format!("https://trello.com/c/{}", card.short_link));
+        }
+        let discord_event = DiscordWebhookEvent {
+            embeds: vec![DiscordEmbed {
+                title: Some(event.action._type),
+                _type: Some(String::from("rich")),
+                description: card_name,
+                url: card_url,
+                color: Some(0x00ff00),
+                fields: vec![],
+                timestamp: Some(event.action.date),
+                author: Self::get_author(event.action.member_creator),
+            }],
+        };
+        self.client
+            .post(&self.endpoint)
+            .json(&discord_event)
+            .send()
+            .await?;
+        Ok(())
+    }
+
+    fn should_skip(event: &WebhookEvent) -> bool {
+        event
+            .action
+            .display
+            .as_ref()
+            .is_some_and(Self::should_skip_action_display)
+    }
+
+    fn should_skip_action_display(action_display: &WebhookActionDisplay) -> bool {
+        action_display
+            .translation_key
+            .as_ref()
+            .is_some_and(|s| Self::should_skip_action_translation_key(s))
+    }
+
+    fn should_skip_action_translation_key(translation_key: &str) -> bool {
+        ["action_moved_card_higher", "action_moved_card_lower"].contains(&translation_key)
```

Comment:
Some constant maybe?
```suggestion
        ACTIONS_MOVE_CARD.contains(&translation_key)
```